### PR TITLE
Fix duplicate call of webhook

### LINKED ISSUE #7702: Webhooks triggered twice
- Gitea version (or commit ref): 1.9.0
- Git version: 2.22.0
- Operating system: Debian Buster / Docker
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant

## Description

Hi,

Since 1.9.0, webhooks are triggered twice when committing via the website.

### PR DESCRIPTION
**Pull request by sapk**
Try to fix #7702

### CODE REVIEW

**Comment by lunny** [comment]
@sapk It seems it still have two webhook event when merged.

image

File: modules/pull/merge.go [hunk 1]
```diff
@@ -310,21 +310,6 @@ func Merge(pr *models.PullRequest, doer *models.User, baseGitRepo *git.Repositor
 		l.PushFront(mergeCommit)
```
Comment (lunny) [inline modules/pull/merge.go]: so it's unnecessary from line 295

Comment (sapk) [reply modules/pull/merge.go]: Yes, I need to also test every merge style to be sure that it trigger the push via post-receive

**Review by lunny — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
Please backport

**Pull request merged**